### DISCUSSION TITLE
Opening a dialog

### DISCUSSION THREAD

**Discussion by gabrielnuss** [gaozhao1989/pyjab#72] [Q&A] [answered]
When I click a button on the java application that opens a dialog, and try to acess the jabdriver again trying to find an element by any means it returns an error called: pyjab.common.exceptions.JABException: ("Java Access Bridge func 'GetAccessibleContextInfo' error", None)

**Comment by gaozhao1989** [comment]
That may caused by incompatible Java Access Bridge, please try with 2.0.1 JAB

**Reply by gabrielnuss** [reply]
Thanks for the answer!
I actually think that this problem was caused because i was trying to reach an element that was in the original window while the new window was in front of it, but i have another question, when i try to declare a new JABDriver for this new window to acess the components it doesn't found it no matter what i do, do you know what this could be?
When i open the application manually and open this new window manually aswell when ir run the python program and try to acess the same window it works and find the window
@gaozhao1989

**Reply by gaozhao1989** [reply]
Wait new Java window load complete before use JABDriver

**Reply by gabrielnuss** [reply]
I tried to wait but doesn't work, the problem is that when i declare a JABDriver to the main window in te application, when i try to connect to any other window in that same application (i'm using Acess bridge explorer to see) it doesn't recognize it not even as a java window, but it is, because if i do the first part manually it recognize the same window fine, please can you help me?

**Reply by gabrielnuss** [reply, marked as answer]
For those who are wondering if i manage to resolve it, my problem was acessing a action button and calling the function "click", when i did this it didn't found the window that opens when i click it, but if instead of calling click function i do this: "actiobButton.sned_text(" ", true)", it works fine even if other window is in front of it.

**Reply by gaozhao1989** [reply]
> For those who are wondering if i manage to resolve it, my problem was acessing a action button and calling the function "click", when i did this it didn't found the window that opens when i click it, but if instead of calling click function i do this: "actiobButton.sned_text(" ", true)", it works fine even if other window is in front of it.

Hi, are you put the Java window in most front of all window before fire click action?
The next question is the component button which you need to click is support accessible action or not?

**Reply by gabrielnuss** [reply]
Hi, so for the first question, no, i didn't try that because i have to run this application as a background application(the java one).
and for the second question, yes, if i open the Acess Bridge explorer, the component button that opens the dialog supports accessible actions, it says that it have one accessible action, that is the click, i hope it answer your questions.
